src/index.js

```diff
@@ -1488,7 +1506,7 @@ EDSCTimeline.propTypes = {
     end: PropTypes.number,
     start: PropTypes.number
   }),
-  // maxDate: PropTypes.number, // maximum date timeline will allow scrolling
+  // MaxDate: PropTypes.number, // maximum date timeline will allow scrolling
   // minDate: PropTypes.number, // minimum date timeline will allow scrolling
```

Comment:
Is it expected that the linter likes `MaxDate` to be PascalCase and `minDate` to be camel case?

Reply:
Should be lower `m`, its an annoying part of one rule. 

But those values aren't even being used anymore, so I'll delete them